### PR TITLE
Add audible feature to cmds/core/ping

### LINKED ISSUE #1445: cmds/core/ping: Add optional audible flag
roadmap.md mentions that for cmds/core/ping the optional -a (audible) flag is desired.

feature description: 
`Include a bell (ASCII 0x07) character in the output when any packet is received.`

### PR DESCRIPTION
**Pull request by TimRots**
Implements optional (-a) audible feature for ping command. Was todo item on roadmap
Fixes #1445

Signed-off-by: Tim Rots <tim.rots@protonmail.ch>

### CODE REVIEW

File: cmds/core/ping/ping.go [hunk 1]
```diff
@@ -158,6 +160,9 @@ func main() {
 		if err != nil {
 			log.Fatalf("ping failed: %v", err)
 		}
+		if *audible {
+			fmt.Print("\a")
```
Comment (rminnich) [inline cmds/core/ping/ping.go]: msg = "\a" + msg
and you can get rid of the fmt.Print

Comment (TimRots) [reply cmds/core/ping/ping.go]: I tried the suggested appending and the and pborman/ansi package. 
However log.Print seems to omit the appended \a. I'm also in favor of getting rid of the fmt.Print but don't really see other options at the moment.

**Comment by hugelgupf** [comment]
If you update this, I'll merge it :)

**Comment by TimRots** [comment]
> If you update this, I'll merge it :)

Done. Thank you Chris.

**Pull request closed without merging**